js/packages/candy-machine-ui/src/Home.tsx

```diff
@@ -604,6 +604,7 @@ const Home = (props: HomeProps) => {
                         ? 'https://api.devnet.solana.com'
                         : rpcUrl
```

Comment:
This check was here to get around the lack of a cluster parameter and can be changed to just rpcUrl instead.

Reply:
thanks, I pushed a commit to just use the rpcUrl now